esmvaltool/recipes/recipe_ocean_bgc.yml

```diff
@@ -186,37 +179,36 @@ diagnostics:
         preprocessor: prep_timeseries_global_volume_average
         mip: Omon
         additional_datasets:
-          - {dataset: WOA,  project: OBS,  type: clim,  version: 2013v2,  start_year: 2000,  end_year: 2000,  tier: 2}
+          - {dataset: WOA,  project: OBS6,  type: clim,  version: 2018,  start_year: 2000,  end_year: 2000,  tier: 2}

```

Comment:
@remi-kazeroni @tomaslovato here - I think we should leave it to use the older OBS type, the problem with fx variables being found in all available mips in the recipe loader test when datasets: all CMIP5 and additional_datasets: all CMIP6 will most probably surface everywhere. The underlying problem is even more hairy: can one actually use such a configuration? The test is dumb and uses monkey patches so that's fine but what if one actually runs such a configuration and eg `volcello` lives indeed in two mips in CMIP6, but none of those for CMIP5

Reply:
plus, I'd feel more comfortable if @ledm has first tested with the updated WOA data then allow the switch, you never know, new WOA might have more whales/sq mile of ocean or something :whale:

Reply:
Let's make a recap ... 
We put back the recipe to use the old WOA2013v2 and leave its update to another PR as previously suggested by @remi-kazeroni .
We complete this PR with the new cmorizer that handles both WOA2013v2 and WOA2018. 
Do we all agree?

Reply:
@remi-kazeroni I think it might be beeter to keep the changes I made to recipe, by adding the title and the use of write plots/netcdf, so we can avoid having a not working recipe in the code.